src/lib/utils.ts

```diff
@@ -415,9 +415,49 @@ export async function findAvailablePort(preferredPort?: number): Promise<number>
 export async function parseCommandLineArgs(args: string[], usage: string) {
   // Process headers
   const headers: Record<string, string> = {}
+  let clientId: string | undefined = undefined
+  let clientSecret: string | undefined = undefined
+  let callbackPath: string | undefined = undefined
+  let scope: string | undefined = undefined
   let i = 0
   while (i < args.length) {
-    if (args[i] === '--header' && i < args.length - 1) {
+    if (args[i] === '--client-id' && i < args.length - 1) {
```

Comment:
I think I'd prefer if there was just a single flag, `--client-info` or `--static-client` that was a JSON value (or, like curl, maybe it can also take a `@/path/to/client-info.json` value?). That just feels more flexible, since you should be effectively able to specify anything in one shot...